src/lib/handlers/auth.js

```diff
@@ -8,6 +13,10 @@ export const authHandlers = {
 	login: async ({ email, password }) => {
 		await signInWithEmailAndPassword(auth, email, password);
 	},
+	resetPassword: async ({ email }) => {
+		let serverResponse = await sendPasswordResetEmail(auth, email);
```

Comment:
Do we need to return the server response here? Why is it different than the other functions?

Reply:
It was for testing something. Thanks for pointing it out

Reply:
Fixed!